src/openzaak/notifications/default.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2022 Dimpact
+from openzaak.components.zaken.models import ZaakObject
+
+
+def handle(message: dict) -> None:
+    resource = message.get("resource")
+    actie = message.get("actie")
+    resource_url = message.get("resourceUrl")
+    kenmerken = message.get("kenmerken", {})
+    objecttype_omschrijving = kenmerken.get("objecttypeOmschrijving")
+    zaken_ids = message.get("zaken", [])
+
+    if resource != "object":
+        return
+
+    if actie == "create":
```

Comment:
Could you use constants for this (and the other two elifs)? https://github.com/maykinmedia/commonground-api-common/blob/f9afc525e1ddceaef0e7252d06a7e6f618ba94de/vng_api_common/constants.py#L254